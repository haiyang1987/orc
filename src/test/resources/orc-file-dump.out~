Structure for target/test/tmp/file.orc
Rows: 21000
Compression: NONE
Type: struct{f: float, d: double}

Statistics:
  Column 0: count: 21000
  Column 1: count: 21000 min: 8.881092071533203E-6 max: 0.9999387264251709 sum: 10551.349509894848
  Column 2: count: 21000 min: 2.3379546596924428E-5 max: 0.9999973837551178 sum: 10486.848248848988

Stripes:
  Stripe: offset: 3 length: 105204 rows: 8751 tail: 44
    Section: column 1 section PRESENT start: 3 length 20
    Section: column 1 section DATA start: 23 length 35004
    Section: column 2 section PRESENT start: 35027 length 20
    Section: column 2 section DATA start: 35047 length 70008
    Section: column 3 section ROW_INDEX start: 105055 length 108
  Stripe: offset: 105207 length: 105208 rows: 8751 tail: 44
    Section: column 1 section PRESENT start: 105207 length 20
    Section: column 1 section DATA start: 105227 length 35004
    Section: column 2 section PRESENT start: 140231 length 20
    Section: column 2 section DATA start: 140251 length 70008
    Section: column 3 section ROW_INDEX start: 210259 length 112
  Stripe: offset: 210415 length: 42151 rows: 3498 tail: 43
    Section: column 1 section PRESENT start: 210415 length 10
    Section: column 1 section DATA start: 210425 length 13992
    Section: column 2 section PRESENT start: 224417 length 10
    Section: column 2 section DATA start: 224427 length 27984
    Section: column 3 section ROW_INDEX start: 252411 length 112
