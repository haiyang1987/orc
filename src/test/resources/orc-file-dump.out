Structure for target/test/tmp/TestFileDump.testDump.orc
Rows: 21000
Compression: ZLIB
Compression size: 10000
Type: struct<f:float,d:double>

Statistics:
  Column 0: count: 21000
  Column 1: count: 21000 min: 8.881092071533203E-6 max: 0.9999387264251709 sum: 10551.349509894848
  Column 2: count: 21000 min: 2.3379546596924428E-5 max: 0.9999973837551178 sum: 10486.848248848992

Stripes:
  Stripe: offset: 3 data: 100425 rows: 9000 tail: 52 index: 98
    Stream: column 0 section ROW_INDEX start: 3 length 10
    Stream: column 1 section ROW_INDEX start: 13 length 41
    Stream: column 2 section ROW_INDEX start: 54 length 47
    Stream: column 1 section PRESENT start: 101 length 12
    Stream: column 1 section DATA start: 113 length 32344
    Stream: column 2 section PRESENT start: 32457 length 12
    Stream: column 2 section DATA start: 32469 length 68057
  Stripe: offset: 100578 data: 100381 rows: 9000 tail: 52 index: 98
    Stream: column 0 section ROW_INDEX start: 100578 length 10
    Stream: column 1 section ROW_INDEX start: 100588 length 42
    Stream: column 2 section ROW_INDEX start: 100630 length 46
    Stream: column 1 section PRESENT start: 100676 length 12
    Stream: column 1 section DATA start: 100688 length 32299
    Stream: column 2 section PRESENT start: 132987 length 12
    Stream: column 2 section DATA start: 132999 length 68058
  Stripe: offset: 201109 data: 33511 rows: 3000 tail: 54 index: 98
    Stream: column 0 section ROW_INDEX start: 201109 length 10
    Stream: column 1 section ROW_INDEX start: 201119 length 41
    Stream: column 2 section ROW_INDEX start: 201160 length 47
    Stream: column 1 section PRESENT start: 201207 length 9
    Stream: column 1 section DATA start: 201216 length 10797
    Stream: column 2 section PRESENT start: 212013 length 9
    Stream: column 2 section DATA start: 212022 length 22696
